readme.md

```diff
@@ -642,7 +642,7 @@ expect(errorWeExceptFor).not.to.be.null;
 
 ```javascript
 it.only("When no product name, it throws error 400", async() => {
-  expect(addNewProduct)).to.eventually.throw(AppError).with.property('code', "InvalidInput");
+  await expect(addNewProduct({})).to.eventually.throw(AppError).with.property('code', "InvalidInput");
```

Comment:
@Userbit Have you checked the need for 'await'?  I don't think it's needed ('eventually' means wait for promise resolve)

Reply:
I use Jest myself, but if I realize it rightly:
Since `expect().to.eventually` is `Promise` per [doc](https://www.chaijs.com/plugins/chai-as-promised/#how-to-use) then it must be `return`. Or in the case of `async` function it is preferable use of `await` statement.

Reply:
@Userbit Always good to learn 👍 🙏 

I also use Jest but tried to mix different frameworks to avoid making it a single-tool guide